app/gui/view/documentation/src/lib.rs

```diff
@@ -180,86 +178,26 @@ impl Model {
-                let closure: Closure<dyn FnMut(MouseEvent)> = Closure::new(move |_: MouseEvent| {
-                    let inner_code = code_block.inner_text();
-                    clipboard::write_text(inner_code);
-                });
-                let callback = closure.as_js_function();
-                match copy_button.add_event_listener_with_callback("click", callback) {
-                    Ok(_) => Some(closure),
-                    Err(e) => {
-                        error!("Unable to add event listener to copy button: {e:?}");
-                        None
-                    }
-                }
-            };
-            create_closures().ok_or(i)
-        });
-        let (closures, errors): (Vec<_>, Vec<_>) = closures.partition(Result::is_ok);
-        let ok_closures = closures.into_iter().filter_map(|t| t.ok()).collect_vec();
-        let err_indices = errors.into_iter().filter_map(|t| t.err()).collect_vec();
-        if !err_indices.is_empty() {
-            error!("Failed to attach listeners to copy buttons with indices: {err_indices:?}.")
+    /// Display the documentation and scroll to the qualified name if needed.
+    fn display_doc(&self, docs: EntryDocumentation) {
+        let anchor = docs.function_name().map(html::anchor_name);
+        let html = html::render(docs);
+        self.inner_dom.dom().set_inner_html(&html);
+        if let Some(anchor) = anchor {
+            if let Some(element) = web::document.get_element_by_id(&anchor) {
+                let offset = element.dyn_ref::<HtmlElement>().map(|e| e.offset_top()).unwrap_or(0);
+                self.inner_dom.dom().set_scroll_top(offset);
+            }
+        } else {
+            self.inner_dom.dom().set_scroll_top(0);
         }
-        self.code_copy_closures.set(ok_closures)
-    }
-
-    /// Receive data, process and present it in the documentation view.
-    fn receive_data(&self, data: &visualization::Data) -> Result<(), visualization::DataError> {
-        let string = match data {
-            visualization::Data::Json { content } => match serde_json::to_string_pretty(&**content)
-            {
-                Ok(string) => string,
-                Err(err) => {
-                    error!(
-                        "Error during documentation vis-data serialization: \
-                        {err:?}"
-                    );
-                    return Err(visualization::DataError::InternalComputationError);
-                }
-            },
-            visualization::Data::Binary =>
-                return Err(visualization::DataError::BinaryNotSupported),
-        };
-        self.display_doc(&string);
-        Ok(())
-    }
-
-    /// Displays the received data in the panel.
-    fn display_doc(&self, content: &str) {
-        self.push_to_dom(String::from(content));
-        self.attach_listeners_to_copy_buttons();
     }
 
     /// Load an HTML file into the documentation view when user is waiting for data to be received.
```

Comment:
This comment was created a long time ago. We should probably add a pivotal task, so this is tracked as part of the current update.